### PR TITLE
Fix name of maven-plugins artifact

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

**Comment by hakonhall** [comment]
Test "storage_bucketdb_gtest_runner_app" failed which I assume is unrelated.

File: maven-plugins/pom.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
         <relativePath>../parent/pom.xml</relativePath>
     </parent>
     <groupId>com.yahoo.vespa</groupId>
-    <artifactId>bundle-plugins</artifactId>
-    <name>bundle-plugins</name>
+    <artifactId>maven-plugins</artifactId>
+    <name>maven-plugins</name>
```
Comment (gjoranv) [inline maven-plugins/pom.xml]: I don't think `name` is needed, so I suggest to remove it.

**Review by gjoranv — APPROVED** [review]

File: maven-plugins/pom.xml (hunk 1 above)
Comment (hakonhall) [reply maven-plugins/pom.xml]: Great, I was hoping you would say that.

Comment (hakonhall) [reply maven-plugins/pom.xml]: Fixed

Comment (gjoranv) [reply maven-plugins/pom.xml]: 👍

**Pull request merged**